src/coreclr/scripts/superpmi_diffs_summarize.py

```diff
@@ -213,7 +214,7 @@ def main(main_args):
             inside_diff = False
             new_lines.append(html_color_diff(cur_diff_lines))
         elif inside_diff:
-            cur_diff_lines.append(line)
```

Comment:
This change was merged separately in #87861, so it won't be part of the commit after I squash-merge.